style.css

```
@@ -1 +1,131 @@
-/* 자유롭게 디자인 해 주세요! */
\ No newline at end of file
+/* 자유롭게 디자인 해 주세요! */
+@font-face {
+  font-family: 'LINESeedKR-Th';
+  src: url('https://fastly.jsdelivr.net/gh/projectnoonnu/noonfonts_11-01@1.0/LINESeedKR-Th.woff2')
+    format('woff2');
+  font-weight: 100;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'LINESeedKR-Rg';
+  src: url('https://fastly.jsdelivr.net/gh/projectnoonnu/noonfonts_11-01@1.0/LINESeedKR-Rg.woff2')
+    format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'LINESeedKR-Bd';
+  src: url('https://fastly.jsdelivr.net/gh/projectnoonnu/noonfonts_11-01@1.0/LINESeedKR-Bd.woff2')
+    format('woff2');
+  font-weight: 700;
+  font-style: normal;
+}
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: linear-gradient(
+    to right,
+    rgb(176, 196, 222),
+    rgb(230, 239, 250)
+  );
+}
+
+.container {
+  margin-top: 120px;
+  width: 360px;
+  height: 500px;
+  border-radius: 20px;
+  padding: 24px;
+  padding-top: 30px;
```

```suggestion
  padding: 30px 24px 24px 24px;
```
이렇게 줄여 쓸 수도 있습니다! 순서대로 상 우 하 좌 (시계방향!)